ontology/EBUCore/ebucore.owl

```diff
@@ -5029,7 +4861,6 @@ bbcconsepts:Organisation rdf:type owl:Class ;
 
 ###  http://www.bbc.co.uk/ontologies/coreconcepts/Person
 bbcconsepts:Person rdf:type owl:Class ;
```

Comment:
Typo: "bbcconsepts" should hava a "c" instead of the "s" and read "bbcconcepts"

Reply:
Usually I'm the one that makes mistakes like this, but this one was in the original document 😄 
We have another Issue where this might be removed from the standard.